spec/models/card_spec.rb

```diff
@@ -1,188 +1,338 @@
-require 'rails_helper'
+require "rails_helper"
 
 describe Card do
-  it 'create card with empty original text' do
-    card = Card.create(original_text: '', translated_text: 'house', user_id: 1,
-                       block_id: 1)
-    expect(card.errors[:original_text]).to include('Необходимо заполнить поле.')
-  end
+  let(:card) { Card.new(original_text: "дом", translated_text: "house",
+    user_id: 1, block_id: 1) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 5, 30 should be on its own line.